services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/rpc/routes.py

```diff
@@ -13,7 +14,12 @@
 def setup_rpc_api_routes(app: FastAPI) -> None:
     async def startup() -> None:
         rpc_server = get_rabbitmq_rpc_server(app)
+        settings: ApplicationSettings = app.state.settings
+
+        rpc_namespace = RPCNamespace.from_entries(
+            {"service": "dy-sidecar", "node_id": f"{settings.DY_SIDECAR_NODE_ID}"}
+        )
         for router in ROUTERS:
-            await rpc_server.register_router(router, DYNAMIC_SIDECAR_RPC_NAMESPACE, app)
+            await rpc_server.register_router(router, rpc_namespace, app)
```

Comment:
Whe dynamic service is unused, is this entry "de-registered" ? If not what is the effect?

Reply:
Queues are removed when the dy-sidecar exists/disconnects.